### PR TITLE
Added support for rightclick event (issue #67)

### PR DESCRIPTION
**Pull request by mwozniczak**

### CODE REVIEW

File: src/graph.js [hunk 1]
```diff
@@ -58,6 +58,8 @@ module.exports = (function () {
 
         this._click = properties.click || undefined;
 
+        this._rightclick = properties.rightclick || undefined;
```
Comment (frewsxcv) [inline src/graph.js]: What do you think about calling it `rightClick`?

Comment (mwozniczak) [inline src/graph.js]: Done.

Comment (frewsxcv) [inline src/graph.js]: Thanks!

**Comment by frewsxcv** [comment]
version 0.6.3 released

**Pull request merged**